Alignment/OfflineValidation/python/TkAlAllInOneTool/alignment.py

```diff
@@ -7,9 +7,10 @@ class Alignment:
     def __init__(self, name, config, runGeomComp = "1"):
         self.condShorts = {
             "TrackerAlignmentErrorExtendedRcd":
-                {"zeroAPE":{"connectString": ("frontier://FrontierProd"
-                                              "/CMS_COND_31X_FROM21X"),
-                            "tagName": "TrackerIdealGeometryErrors210_mc",
+                {"zeroAPE":{"connectString": ("frontier://PromptProd"
```

Comment:
Changing this line was one of the main fixes to get the extended alignment parameter errors working. I had a look at the recent global tags and there TrackerAlignmentErrorExtendedRcd has frontier://PromptProd/CMS_COND_ALIGN_000 as PFN
Since I am new to alignment I am not sure what else to use.